INFO 2019-06-12 18:34:04,838 autoreload 576 Watching for file changes with StatReloader
INFO 2019-06-12 18:34:25,586 autoreload 203 /home/pyvip/code/mysite/mysite/settings.py changed, reloading.
INFO 2019-06-12 18:34:26,663 autoreload 576 Watching for file changes with StatReloader
INFO 2019-06-12 18:35:18,700 autoreload 203 /home/pyvip/code/mysite/mysite/settings.py changed, reloading.
INFO 2019-08-16 01:51:54,298 basehttp 154 "GET /news HTTP/1.1" 301 0
ERROR 2019-08-16 01:51:54,756 log 228 Internal Server Error: /news/
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/pyvip/code/mysite/apps/news/views.py", line 54, in get
    news = news_queryset.filter(is_delete=False, tag_id=tag_id) or news_queryset.filter(is_delete=False)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_delete' into field. Choices are: author, author, author_id, banner, clicks, comments, content, create_time, digest, hotnews, id, image_url, is_deleted, tag, tag_id, tag_name, title, update_time
ERROR 2019-08-16 01:51:54,990 basehttp 154 "GET /news/ HTTP/1.1" 500 138529
WARNING 2019-08-16 01:51:57,191 log 228 Not Found: /favicon.ico
WARNING 2019-08-16 01:51:57,193 basehttp 154 "GET /favicon.ico HTTP/1.1" 404 3192
ERROR 2019-08-16 01:52:15,924 log 228 Internal Server Error: /
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/pyvip/code/mysite/apps/news/views.py", line 22, in index
    tags = Tag.objects.only('id', 'name').filter(is_delete=False)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_delete' into field. Choices are: create_time, id, is_deleted, name, news, update_time
ERROR 2019-08-16 01:52:15,930 basehttp 154 "GET / HTTP/1.1" 500 98488
INFO 2019-08-16 01:52:35,609 basehttp 154 "GET / HTTP/1.1" 200 11553
INFO 2019-08-16 01:52:35,755 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-16 01:52:35,756 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-16 01:52:35,760 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-16 01:52:35,871 basehttp 154 "GET /static/css/news/index.css HTTP/1.1" 200 5500
INFO 2019-08-16 01:52:35,960 basehttp 154 "GET /static/js/news/index.js HTTP/1.1" 200 4540
INFO 2019-08-16 01:52:35,965 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-16 01:52:35,965 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-16 01:52:36,059 basehttp 154 "GET /static/images/ui.png HTTP/1.1" 200 108619
INFO 2019-08-16 01:52:36,173 basehttp 154 "GET /static/images/python_gui.jpg HTTP/1.1" 200 24129
INFO 2019-08-16 01:52:36,176 basehttp 154 "GET /static/images/linux.jpg HTTP/1.1" 200 62189
INFO 2019-08-16 01:52:36,786 basehttp 154 "GET /static/images/zimeiti.png HTTP/1.1" 200 112767
INFO 2019-08-16 01:52:36,787 basehttp 154 "GET /static/images/python_advanced.jpg HTTP/1.1" 200 120059
INFO 2019-08-16 01:52:37,325 basehttp 154 "GET /static/images/wechat2.jpg HTTP/1.1" 200 41049
INFO 2019-08-16 01:52:37,733 basehttp 154 "GET /static/images/english.jpg HTTP/1.1" 200 137605
INFO 2019-08-16 01:52:38,039 basehttp 154 "GET /static/images/jichujiaochen.jpeg HTTP/1.1" 200 11286
WARNING 2019-08-16 01:52:38,363 log 228 Not Found: /index.html
WARNING 2019-08-16 01:52:38,372 basehttp 154 "GET /index.html HTTP/1.1" 404 3189
INFO 2019-08-16 01:52:38,456 basehttp 154 "GET /static/images/youxi.png HTTP/1.1" 200 303104
INFO 2019-08-16 01:52:38,555 basehttp 154 "GET /static/images/dianshang.png HTTP/1.1" 200 303104
INFO 2019-08-16 01:52:38,567 basehttp 154 "GET /static/images/python_web.jpg HTTP/1.1" 200 45717
ERROR 2019-08-16 01:52:38,613 log 228 Internal Server Error: /news/
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/pyvip/code/mysite/apps/news/views.py", line 54, in get
    news = news_queryset.filter(is_delete=False, tag_id=tag_id) or news_queryset.filter(is_delete=False)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_delete' into field. Choices are: author, author, author_id, banner, clicks, comments, content, create_time, digest, hotnews, id, image_url, is_deleted, tag, tag_id, tag_name, title, update_time
ERROR 2019-08-16 01:52:38,625 basehttp 154 "GET /news/?tag=0&page=1 HTTP/1.1" 500 15925
INFO 2019-08-16 01:52:40,023 basehttp 154 "GET /static/images/python_web.jpg HTTP/1.1" 200 45717
INFO 2019-08-16 01:52:40,115 basehttp 154 "GET /static/images/python_advanced.jpg HTTP/1.1" 200 120059
INFO 2019-08-16 01:52:40,116 basehttp 154 "GET /static/images/english.jpg HTTP/1.1" 200 137605
ERROR 2019-08-16 01:52:40,195 log 228 Internal Server Error: /news/
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/pyvip/code/mysite/apps/news/views.py", line 54, in get
    news = news_queryset.filter(is_delete=False, tag_id=tag_id) or news_queryset.filter(is_delete=False)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_delete' into field. Choices are: author, author, author_id, banner, clicks, comments, content, create_time, digest, hotnews, id, image_url, is_deleted, tag, tag_id, tag_name, title, update_time
ERROR 2019-08-16 01:52:40,206 basehttp 154 "GET /news/?tag=0&page=1 HTTP/1.1" 500 15925
INFO 2019-08-16 01:52:42,093 basehttp 154 "GET /static/images/dianshang.png HTTP/1.1" 200 342554
INFO 2019-08-16 01:52:44,899 basehttp 154 "GET /static/images/youxi.png HTTP/1.1" 200 447834
ERROR 2019-08-16 01:52:48,281 log 228 Internal Server Error: /news/
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/pyvip/code/mysite/apps/news/views.py", line 54, in get
    news = news_queryset.filter(is_delete=False, tag_id=tag_id) or news_queryset.filter(is_delete=False)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_delete' into field. Choices are: author, author, author_id, banner, clicks, comments, content, create_time, digest, hotnews, id, image_url, is_deleted, tag, tag_id, tag_name, title, update_time
ERROR 2019-08-16 01:52:49,162 basehttp 154 "GET /news/ HTTP/1.1" 500 138529
ERROR 2019-08-16 01:54:22,291 log 228 Internal Server Error: /news/
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/pyvip/code/mysite/apps/news/views.py", line 54, in get
    news = news_queryset.filter(is_delete=False, tag_id=tag_id) or news_queryset.filter(is_delete=False)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_delete' into field. Choices are: author, author, author_id, banner, clicks, comments, content, create_time, digest, hotnews, id, image_url, is_deleted, tag, tag_id, tag_name, title, update_time
ERROR 2019-08-16 01:54:22,683 basehttp 154 "GET /news/ HTTP/1.1" 500 138790
ERROR 2019-08-16 02:03:18,829 log 228 Internal Server Error: /news/
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/pyvip/code/mysite/apps/news/views.py", line 54, in get
    news = news_queryset.filter(is_delete=False, tag_id=tag_id) or news_queryset.filter(is_delete=False)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_delete' into field. Choices are: author, author, author_id, banner, clicks, comments, content, create_time, digest, hotnews, id, image_url, is_deleted, tag, tag_id, tag_name, title, update_time
ERROR 2019-08-16 02:03:19,088 basehttp 154 "GET /news/ HTTP/1.1" 500 138790
ERROR 2019-08-16 02:03:22,631 log 228 Internal Server Error: /news/
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/pyvip/code/mysite/apps/news/views.py", line 54, in get
    news = news_queryset.filter(is_delete=False, tag_id=tag_id) or news_queryset.filter(is_delete=False)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_delete' into field. Choices are: author, author, author_id, banner, clicks, comments, content, create_time, digest, hotnews, id, image_url, is_deleted, tag, tag_id, tag_name, title, update_time
ERROR 2019-08-16 02:03:23,560 basehttp 154 "GET /news/ HTTP/1.1" 500 138790
ERROR 2019-08-16 02:04:23,426 log 228 Internal Server Error: /news/
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/pyvip/code/mysite/apps/news/views.py", line 54, in get
    news = news_queryset.filter(is_delete=False, tag_id=tag_id) or news_queryset.filter(is_delete=False)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 844, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/query.py", line 862, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1263, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1287, in _add_q
    split_subq=split_subq,
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1164, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1028, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.6/dist-packages/django/db/models/sql/query.py", line 1389, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_delete' into field. Choices are: author, author, author_id, banner, clicks, comments, content, create_time, digest, hotnews, id, image_url, is_deleted, tag, tag_id, tag_name, title, update_time
ERROR 2019-08-16 02:04:24,301 basehttp 154 "GET /news/ HTTP/1.1" 500 138790
INFO 2019-08-16 02:06:47,092 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:07:45,640 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:07:47,726 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:07:48,665 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:07:48,884 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:07:49,088 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:07:49,300 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:07:49,492 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:07:49,828 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:15:55,625 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:15:57,357 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:16:08,392 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:16:09,314 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:16:09,733 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:16:10,059 basehttp 154 "GET /news/ HTTP/1.1" 200 4869
INFO 2019-08-16 02:16:31,276 basehttp 154 "GET / HTTP/1.1" 200 11549
INFO 2019-08-16 02:16:31,439 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-16 02:16:31,445 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-16 02:16:31,446 basehttp 154 "GET /static/css/news/index.css HTTP/1.1" 200 5500
INFO 2019-08-16 02:16:31,472 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-16 02:16:31,639 basehttp 154 "GET /static/images/ui.png HTTP/1.1" 200 108619
INFO 2019-08-16 02:16:31,641 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-16 02:16:31,643 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
WARNING 2019-08-16 02:16:31,661 basehttp 154 "GET /static/js/news/index.jss HTTP/1.1" 404 1675
INFO 2019-08-16 02:16:31,856 basehttp 154 "GET /static/images/python_gui.jpg HTTP/1.1" 200 24129
INFO 2019-08-16 02:16:32,055 basehttp 154 "GET /static/images/zimeiti.png HTTP/1.1" 200 112767
INFO 2019-08-16 02:16:32,056 basehttp 154 "GET /static/images/english.jpg HTTP/1.1" 200 137605
WARNING 2019-08-16 02:16:32,272 basehttp 154 "GET /static/js/news/index.jss HTTP/1.1" 404 1675
INFO 2019-08-16 02:16:32,474 basehttp 154 "GET /static/images/linux.jpg HTTP/1.1" 200 62189
INFO 2019-08-16 02:16:33,409 basehttp 154 "GET /static/images/python_advanced.jpg HTTP/1.1" 200 120059
INFO 2019-08-16 02:16:33,617 basehttp 154 "GET /static/images/jichujiaochen.jpeg HTTP/1.1" 200 11286
INFO 2019-08-16 02:16:33,929 basehttp 154 "GET /static/images/python_web.jpg HTTP/1.1" 200 45717
INFO 2019-08-16 02:16:33,931 basehttp 154 "GET /static/images/wechat2.jpg HTTP/1.1" 200 41049
INFO 2019-08-16 02:16:35,175 basehttp 154 "GET /static/images/dianshang.png HTTP/1.1" 200 342554
INFO 2019-08-16 02:16:37,359 basehttp 154 "GET /static/images/youxi.png HTTP/1.1" 200 447834
INFO 2019-08-16 02:18:07,088 basehttp 154 "GET / HTTP/1.1" 200 11548
INFO 2019-08-16 02:18:07,179 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-16 02:18:07,185 basehttp 154 "GET /static/css/news/index.css HTTP/1.1" 200 5500
INFO 2019-08-16 02:18:07,186 basehttp 154 "GET /static/images/ui.png HTTP/1.1" 200 108619
INFO 2019-08-16 02:18:07,188 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-16 02:18:07,189 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-16 02:18:07,386 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-16 02:18:07,389 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-16 02:18:07,405 basehttp 154 "GET /static/images/zimeiti.png HTTP/1.1" 200 112767
INFO 2019-08-16 02:18:07,596 basehttp 154 "GET /static/js/news/index.js HTTP/1.1" 200 5929
INFO 2019-08-16 02:18:07,599 basehttp 154 "GET /static/images/english.jpg HTTP/1.1" 200 137605
INFO 2019-08-16 02:18:07,800 basehttp 154 "GET /static/images/python_gui.jpg HTTP/1.1" 200 24129
INFO 2019-08-16 02:18:08,251 basehttp 154 "GET /news/?tag=0&page=1 HTTP/1.1" 200 6142
INFO 2019-08-16 02:18:08,427 basehttp 154 "GET /static/images/linux.jpg HTTP/1.1" 200 62189
INFO 2019-08-16 02:18:09,153 basehttp 154 "GET /static/images/python_advanced.jpg HTTP/1.1" 200 120059
INFO 2019-08-16 02:18:09,467 basehttp 154 "GET /static/images/jichujiaochen.jpeg HTTP/1.1" 200 11286
INFO 2019-08-16 02:18:09,569 basehttp 154 "GET /static/images/python_web.jpg HTTP/1.1" 200 45717
INFO 2019-08-16 02:18:09,777 basehttp 154 "GET /static/images/wechat2.jpg HTTP/1.1" 200 41049
WARNING 2019-08-16 02:18:10,096 log 228 Not Found: /media/python_web.jpg
WARNING 2019-08-16 02:18:10,097 basehttp 154 "GET /media/python_web.jpg HTTP/1.1" 404 1788
INFO 2019-08-16 02:18:10,919 basehttp 154 "GET /static/images/dianshang.png HTTP/1.1" 200 342554
INFO 2019-08-16 02:18:13,208 basehttp 154 "GET /static/images/youxi.png HTTP/1.1" 200 447834
INFO 2019-08-16 02:18:48,711 basehttp 154 "GET / HTTP/1.1" 200 11548
INFO 2019-08-16 02:18:48,792 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-16 02:18:48,796 basehttp 154 "GET /static/css/news/index.css HTTP/1.1" 200 5500
INFO 2019-08-16 02:18:48,799 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-16 02:18:48,800 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-16 02:18:49,007 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-16 02:18:49,015 basehttp 154 "GET /static/js/news/index.js HTTP/1.1" 200 5929
INFO 2019-08-16 02:18:49,016 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-16 02:18:49,108 basehttp 154 "GET /static/images/ui.png HTTP/1.1" 200 108619
INFO 2019-08-16 02:18:49,211 basehttp 154 "GET /static/images/zimeiti.png HTTP/1.1" 200 112767
INFO 2019-08-16 02:18:49,215 basehttp 154 "GET /static/images/python_gui.jpg HTTP/1.1" 200 24129
INFO 2019-08-16 02:18:49,415 basehttp 154 "GET /static/images/english.jpg HTTP/1.1" 200 137605
INFO 2019-08-16 02:18:49,684 basehttp 154 "GET /news/?tag=0&page=1 HTTP/1.1" 200 6142
INFO 2019-08-16 02:18:49,833 basehttp 154 "GET /static/images/linux.jpg HTTP/1.1" 200 62189
INFO 2019-08-16 02:18:50,874 basehttp 154 "GET /static/images/python_advanced.jpg HTTP/1.1" 200 120059
INFO 2019-08-16 02:18:50,979 basehttp 154 "GET /static/images/jichujiaochen.jpeg HTTP/1.1" 200 11286
INFO 2019-08-16 02:18:51,291 basehttp 154 "GET /static/images/wechat2.jpg HTTP/1.1" 200 41049
INFO 2019-08-16 02:18:51,294 basehttp 154 "GET /static/images/python_web.jpg HTTP/1.1" 200 45717
WARNING 2019-08-16 02:18:51,715 log 228 Not Found: /media/python_web.jpg
WARNING 2019-08-16 02:18:51,716 basehttp 154 "GET /media/python_web.jpg HTTP/1.1" 404 1788
INFO 2019-08-16 02:18:52,430 basehttp 154 "GET /static/images/dianshang.png HTTP/1.1" 200 342554
INFO 2019-08-16 02:18:54,822 basehttp 154 "GET /static/images/youxi.png HTTP/1.1" 200 447834
INFO 2019-08-16 02:19:10,753 basehttp 154 "GET / HTTP/1.1" 200 11548
INFO 2019-08-16 02:19:10,841 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-16 02:19:10,844 basehttp 154 "GET /static/css/news/index.css HTTP/1.1" 200 5500
INFO 2019-08-16 02:19:10,847 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-16 02:19:10,848 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-16 02:19:11,059 basehttp 154 "GET /static/js/news/index.js HTTP/1.1" 200 5929
INFO 2019-08-16 02:19:11,060 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-16 02:19:11,061 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-16 02:19:11,153 basehttp 154 "GET /static/images/ui.png HTTP/1.1" 200 108619
INFO 2019-08-16 02:19:11,257 basehttp 154 "GET /static/images/python_gui.jpg HTTP/1.1" 200 24129
INFO 2019-08-16 02:19:11,261 basehttp 154 "GET /static/images/python_advanced.jpg HTTP/1.1" 200 120059
INFO 2019-08-16 02:19:11,264 basehttp 154 "GET /static/images/english.jpg HTTP/1.1" 200 137605
INFO 2019-08-16 02:19:11,674 basehttp 154 "GET /static/images/jichujiaochen.jpeg HTTP/1.1" 200 11286
INFO 2019-08-16 02:19:12,025 basehttp 154 "GET /news/?tag=0&page=1 HTTP/1.1" 200 6142
INFO 2019-08-16 02:19:12,194 basehttp 154 "GET /static/images/zimeiti.png HTTP/1.1" 200 112767
INFO 2019-08-16 02:19:12,817 basehttp 154 "GET /static/images/linux.jpg HTTP/1.1" 200 62189
INFO 2019-08-16 02:19:13,441 basehttp 154 "GET /static/images/python_web.jpg HTTP/1.1" 200 45717
INFO 2019-08-16 02:19:13,752 basehttp 154 "GET /static/images/wechat2.jpg HTTP/1.1" 200 41049
INFO 2019-08-16 02:19:13,962 basehttp 154 "GET /media/python_web.jpg HTTP/1.1" 200 45717
INFO 2019-08-16 02:19:14,480 basehttp 154 "GET /static/images/dianshang.png HTTP/1.1" 200 342554
INFO 2019-08-16 02:19:16,871 basehttp 154 "GET /static/images/youxi.png HTTP/1.1" 200 447834
INFO 2019-08-16 02:19:48,006 basehttp 154 "GET /news/?tag=0&page=2 HTTP/1.1" 200 5662
INFO 2019-08-17 14:10:33,459 basehttp 154 "GET / HTTP/1.1" 200 11548
INFO 2019-08-17 14:10:33,706 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-17 14:10:33,708 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-17 14:10:33,712 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-17 14:10:33,717 basehttp 154 "GET /static/css/news/index.css HTTP/1.1" 200 5500
INFO 2019-08-17 14:10:33,900 basehttp 154 "GET /static/images/ui.png HTTP/1.1" 200 108619
INFO 2019-08-17 14:10:33,929 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-17 14:10:33,908 basehttp 154 "GET /static/js/news/index.js HTTP/1.1" 200 5929
INFO 2019-08-17 14:10:33,930 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-17 14:10:34,127 basehttp 154 "GET /static/images/wechat2.jpg HTTP/1.1" 200 41049
INFO 2019-08-17 14:10:34,316 basehttp 154 "GET /static/images/python_advanced.jpg HTTP/1.1" 200 120059
INFO 2019-08-17 14:10:34,317 basehttp 154 "GET /static/images/english.jpg HTTP/1.1" 200 137605
INFO 2019-08-17 14:10:34,944 basehttp 154 "GET /static/images/python_gui.jpg HTTP/1.1" 200 24129
INFO 2019-08-17 14:10:35,359 basehttp 154 "GET /static/images/jichujiaochen.jpeg HTTP/1.1" 200 11286
INFO 2019-08-17 14:10:35,672 basehttp 154 "GET /static/images/linux.jpg HTTP/1.1" 200 62189
INFO 2019-08-17 14:10:35,876 basehttp 154 "GET /static/images/zimeiti.png HTTP/1.1" 200 112767
INFO 2019-08-17 14:10:36,304 basehttp 154 "GET /users/register/ HTTP/1.1" 200 4620
INFO 2019-08-17 14:10:36,501 basehttp 154 "GET /static/images/dianshang.png HTTP/1.1" 200 303104
INFO 2019-08-17 14:10:36,501 basehttp 154 "GET /static/images/youxi.png HTTP/1.1" 200 335872
INFO 2019-08-17 14:10:36,523 basehttp 154 "GET /static/css/users/auth.css HTTP/1.1" 200 3286
INFO 2019-08-17 14:10:36,524 basehttp 154 "GET /static/images/python_web.jpg HTTP/1.1" 200 45717
INFO 2019-08-17 14:10:36,535 basehttp 154 "GET /static/js/users/register.js HTTP/1.1" 200 7978
INFO 2019-08-17 14:10:36,572 views 36 Image code:ACMH
INFO 2019-08-17 14:10:36,588 basehttp 154 "GET /image_code/ HTTP/1.1" 200 3381
INFO 2019-08-17 14:11:42,253 basehttp 154 "GET /users/register/ HTTP/1.1" 200 4620
INFO 2019-08-17 14:11:42,349 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-17 14:11:42,354 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-17 14:11:42,465 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-17 14:11:42,469 basehttp 154 "GET /static/css/users/auth.css HTTP/1.1" 200 3286
INFO 2019-08-17 14:11:42,480 views 36 Image code:FV5U
INFO 2019-08-17 14:11:42,483 basehttp 154 "GET /image_code/ HTTP/1.1" 200 3308
INFO 2019-08-17 14:11:42,557 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-17 14:11:42,559 basehttp 154 "GET /static/js/users/register.js HTTP/1.1" 200 7978
INFO 2019-08-17 14:11:42,563 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-17 14:12:18,502 basehttp 154 "GET / HTTP/1.1" 200 11548
INFO 2019-08-17 14:12:18,686 basehttp 154 "GET /static/images/linux.jpg HTTP/1.1" 200 62189
INFO 2019-08-17 14:12:18,782 basehttp 154 "GET /static/images/python_web.jpg HTTP/1.1" 200 45717
INFO 2019-08-17 14:12:18,884 basehttp 154 "GET /static/images/zimeiti.png HTTP/1.1" 200 112767
INFO 2019-08-17 14:12:19,092 basehttp 154 "GET /static/images/english.jpg HTTP/1.1" 200 137605
INFO 2019-08-17 14:12:22,212 basehttp 154 "GET /static/images/dianshang.png HTTP/1.1" 200 342554
INFO 2019-08-17 14:12:24,507 basehttp 154 "GET /static/images/youxi.png HTTP/1.1" 200 447834
INFO 2019-08-17 14:12:28,886 basehttp 154 "GET /users/register/ HTTP/1.1" 200 4620
INFO 2019-08-17 14:12:29,098 views 36 Image code:3YYS
INFO 2019-08-17 14:12:29,101 basehttp 154 "GET /image_code/ HTTP/1.1" 200 2986
INFO 2019-08-17 14:15:35,757 basehttp 154 "GET /users/register/ HTTP/1.1" 200 4620
INFO 2019-08-17 14:15:35,904 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-17 14:15:35,913 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-17 14:15:35,919 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-17 14:15:35,928 basehttp 154 "GET /static/css/users/auth.css HTTP/1.1" 200 3286
INFO 2019-08-17 14:15:35,959 views 36 Image code:8JOG
INFO 2019-08-17 14:15:35,962 basehttp 154 "GET /image_code/ HTTP/1.1" 200 2964
INFO 2019-08-17 14:15:36,005 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-17 14:15:36,107 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-17 14:15:36,110 basehttp 154 "GET /static/js/users/register.js HTTP/1.1" 200 7978
INFO 2019-08-17 14:15:40,084 basehttp 154 "GET /users/register/ HTTP/1.1" 200 4620
INFO 2019-08-17 14:15:40,424 views 36 Image code:EWKS
INFO 2019-08-17 14:15:40,432 basehttp 154 "GET /image_code/ HTTP/1.1" 200 3082
WARNING 2019-08-17 14:16:19,496 log 228 Not Found: /favicon.ico
WARNING 2019-08-17 14:16:19,503 basehttp 154 "GET /favicon.ico HTTP/1.1" 404 3192
WARNING 2019-08-17 14:16:19,502 log 228 Not Found: /favicon.ico
WARNING 2019-08-17 14:16:19,505 basehttp 154 "GET /favicon.ico HTTP/1.1" 404 3192
INFO 2019-08-17 14:16:21,159 basehttp 154 "GET /users/register/ HTTP/1.1" 200 4620
INFO 2019-08-17 14:16:21,256 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-17 14:16:21,363 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-17 14:16:21,373 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-17 14:16:21,381 basehttp 154 "GET /static/css/users/auth.css HTTP/1.1" 200 3286
INFO 2019-08-17 14:16:21,391 views 36 Image code:GW8A
INFO 2019-08-17 14:16:21,394 basehttp 154 "GET /image_code/ HTTP/1.1" 200 3121
INFO 2019-08-17 14:16:21,464 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-17 14:16:21,471 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-17 14:16:21,570 basehttp 154 "GET /static/js/users/register.js HTTP/1.1" 200 7978
INFO 2019-08-17 14:16:24,798 basehttp 154 "GET /users/register/ HTTP/1.1" 200 4620
INFO 2019-08-17 14:16:24,896 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-17 14:16:24,899 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-17 14:16:24,901 basehttp 154 "GET /static/css/users/auth.css HTTP/1.1" 200 3286
INFO 2019-08-17 14:16:24,902 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-17 14:16:25,006 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-17 14:16:25,048 views 36 Image code:XAMQ
INFO 2019-08-17 14:16:25,066 basehttp 154 "GET /image_code/ HTTP/1.1" 200 3172
INFO 2019-08-17 14:16:25,103 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-17 14:16:25,105 basehttp 154 "GET /static/js/users/register.js HTTP/1.1" 200 7978
INFO 2019-08-17 14:16:37,908 basehttp 154 "GET /username/JaneZhou/ HTTP/1.1" 200 74
INFO 2019-08-17 14:16:53,088 basehttp 154 "GET /username/CESHI/ HTTP/1.1" 200 71
INFO 2019-08-17 14:17:00,265 basehttp 154 "GET /mobile/13888888888/ HTTP/1.1" 200 75
ERROR 2019-08-17 14:17:06,288 views 143 发送短信验证码[失败][moblie: 13888888888 sms_code: 7031]
INFO 2019-08-17 14:17:06,289 basehttp 154 "POST /sms_code/ HTTP/1.1" 200 99
INFO 2019-08-17 14:17:34,587 basehttp 154 "GET /mobile/13888888888/ HTTP/1.1" 200 75
ERROR 2019-08-17 14:17:35,406 views 143 发送短信验证码[失败][moblie: 13888888888 sms_code: 3906]
INFO 2019-08-17 14:17:35,407 basehttp 154 "POST /sms_code/ HTTP/1.1" 200 99
INFO 2019-08-17 14:17:41,671 basehttp 154 "POST /users/register/ HTTP/1.1" 200 88
INFO 2019-08-17 14:18:24,626 basehttp 154 "GET /mobile/18819290253/ HTTP/1.1" 200 75
ERROR 2019-08-17 14:18:25,550 views 143 发送短信验证码[失败][moblie: 18819290253 sms_code: 6585]
INFO 2019-08-17 14:18:25,552 basehttp 154 "POST /sms_code/ HTTP/1.1" 200 99
ERROR 2019-08-17 14:18:32,934 views 143 发送短信验证码[失败][moblie: 18819290253 sms_code: 5714]
INFO 2019-08-17 14:18:32,936 basehttp 154 "POST /sms_code/ HTTP/1.1" 200 99
INFO 2019-08-17 14:18:36,903 basehttp 154 "GET /mobile/18819290253/ HTTP/1.1" 200 75
INFO 2019-08-17 14:18:38,667 basehttp 154 "GET /mobile/18819290253/ HTTP/1.1" 200 75
INFO 2019-08-17 14:18:51,356 basehttp 154 "GET /mobile/13026323177/ HTTP/1.1" 200 75
INFO 2019-08-17 14:18:53,214 views 141 发送短信验证码[正常][mobile: 13026323177 sms_code: 3973]
INFO 2019-08-17 14:18:53,218 basehttp 154 "POST /sms_code/ HTTP/1.1" 200 96
INFO 2019-08-17 14:19:07,067 basehttp 154 "POST /users/register/ HTTP/1.1" 200 96
INFO 2019-08-17 14:19:08,721 basehttp 154 "GET /users/login/ HTTP/1.1" 200 3942
INFO 2019-08-17 14:19:08,815 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-17 14:19:08,819 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-17 14:19:08,821 basehttp 154 "GET /static/css/users/auth.css HTTP/1.1" 200 3286
INFO 2019-08-17 14:19:08,822 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-17 14:19:08,920 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-17 14:19:08,922 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-17 14:19:09,030 basehttp 154 "GET /static/js/users/login.js HTTP/1.1" 200 2143
INFO 2019-08-17 14:20:01,670 basehttp 154 "GET /users/register/ HTTP/1.1" 200 4620
INFO 2019-08-17 14:20:01,767 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-17 14:20:01,774 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-17 14:20:01,774 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-17 14:20:01,773 basehttp 154 "GET /static/css/users/auth.css HTTP/1.1" 200 3286
INFO 2019-08-17 14:20:01,871 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-17 14:20:01,887 views 36 Image code:ICY7
INFO 2019-08-17 14:20:01,889 basehttp 154 "GET /image_code/ HTTP/1.1" 200 3027
INFO 2019-08-17 14:20:01,983 basehttp 154 "GET /static/js/users/register.js HTTP/1.1" 200 7978
INFO 2019-08-17 14:20:01,983 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-17 14:20:21,323 basehttp 154 "GET /users/register/ HTTP/1.1" 200 4620
INFO 2019-08-17 14:20:21,540 views 36 Image code:ZR7W
INFO 2019-08-17 14:20:21,542 basehttp 154 "GET /image_code/ HTTP/1.1" 200 3443
INFO 2019-08-17 14:20:26,215 basehttp 154 "GET /username/ceshi2/ HTTP/1.1" 200 72
INFO 2019-08-17 14:20:33,390 basehttp 154 "GET /mobile/13026323177/ HTTP/1.1" 200 75
INFO 2019-08-17 14:20:39,726 views 141 发送短信验证码[正常][mobile: 13026323177 sms_code: 0198]
INFO 2019-08-17 14:20:39,727 basehttp 154 "POST /sms_code/ HTTP/1.1" 200 96
INFO 2019-08-17 14:20:51,481 basehttp 154 "POST /users/register/ HTTP/1.1" 200 96
INFO 2019-08-17 14:20:53,152 basehttp 154 "GET /users/login/ HTTP/1.1" 200 3942
INFO 2019-08-19 01:00:35,701 basehttp 154 "GET / HTTP/1.1" 200 11548
INFO 2019-08-19 01:00:35,928 basehttp 154 "GET /static/css/base/reset.css HTTP/1.1" 200 1427
INFO 2019-08-19 01:00:35,930 basehttp 154 "GET /static/css/base/common.css HTTP/1.1" 200 3498
INFO 2019-08-19 01:00:35,935 basehttp 154 "GET /static/css/base/side.css HTTP/1.1" 200 4429
INFO 2019-08-19 01:00:35,940 basehttp 154 "GET /static/css/news/index.css HTTP/1.1" 200 5500
INFO 2019-08-19 01:00:36,140 basehttp 154 "GET /static/images/ui.png HTTP/1.1" 200 108619
INFO 2019-08-19 01:00:36,150 basehttp 154 "GET /static/js/news/index.js HTTP/1.1" 200 5929
INFO 2019-08-19 01:00:36,154 basehttp 154 "GET /static/js/commons.js HTTP/1.1" 200 1227
INFO 2019-08-19 01:00:36,156 basehttp 154 "GET /static/js/base/message.js HTTP/1.1" 200 3570
INFO 2019-08-19 01:00:36,345 basehttp 154 "GET /static/images/jichujiaochen.jpeg HTTP/1.1" 200 11286
INFO 2019-08-19 01:00:36,346 basehttp 154 "GET /static/images/wechat2.jpg HTTP/1.1" 200 41049
INFO 2019-08-19 01:00:36,547 basehttp 154 "GET /static/images/english.jpg HTTP/1.1" 200 137605
INFO 2019-08-19 01:00:36,655 basehttp 154 "GET /static/images/python_gui.jpg HTTP/1.1" 200 24129
INFO 2019-08-19 01:00:37,275 basehttp 154 "GET /static/images/zimeiti.png HTTP/1.1" 200 112767
INFO 2019-08-19 01:00:37,276 basehttp 154 "GET /static/images/python_advanced.jpg HTTP/1.1" 200 120059
INFO 2019-08-19 01:00:37,911 basehttp 154 "GET /static/images/linux.jpg HTTP/1.1" 200 62189
INFO 2019-08-19 01:00:38,839 basehttp 154 "GET /static/images/python_web.jpg HTTP/1.1" 200 45717
INFO 2019-08-19 01:00:39,667 basehttp 154 "GET /static/images/dianshang.png HTTP/1.1" 200 342554
INFO 2019-08-19 01:00:41,748 basehttp 154 "GET /static/images/youxi.png HTTP/1.1" 200 447834
WARNING 2019-08-19 01:09:46,550 log 228 Not Found: /favicon.ico
WARNING 2019-08-19 01:09:46,554 basehttp 154 "GET /favicon.ico HTTP/1.1" 404 3192
